lms/djangoapps/teams/static/teams/js/views/topic_teams.js

```diff
@@ -33,7 +32,7 @@
                 return this.context.userInfo.staff
                     || this.context.userInfo.privileged
                     || (!TeamUtils.isInstructorManagedTopic(this.model.attributes.type)
-                        && this.myTeamsCollection.length === 0);
+                        && this.collection.length === 0);
```

Comment:
i think the collection for the topic_teams view all of the teams in the topic, right? this looks to me like it's checking if there are any teams in the topic?

Reply:
The impression I got from the code and what was in there, was that it was that `this.collection` was the set of teams in the teamset to which the user belonged.

In retrospect, I will double check that and see if I need to add more filtering.